clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsOptions.java

```diff
@@ -59,7 +59,7 @@ public CreateTopicsOptions validateOnly(boolean validateOnly) {
     /**
      * Return true if the request should be validated without creating the topic.
      */
-    public boolean validateOnly() {
+    public boolean isValidateOnly() {
```

Comment:
This name does read a little awkwardly with the `is` prefix. Maybe `shouldValidateOnly`?